src/ui/modals/confirm_queue_clear.rs

```diff
@@ -60,51 +62,45 @@ impl ConfirmQueueClearModal<'_> {
 
 impl Modal for ConfirmQueueClearModal<'_> {
     fn render(&mut self, frame: &mut Frame, app: &mut AppContext) -> Result<()> {
+        let popup_area = frame.area().centered_exact(45, 6);
+        frame.render_widget(Clear, popup_area);
+
+        if let Some(bg_color) = app.config.theme.modal_background_color {
+            frame.render_widget(Block::default().style(Style::default().bg(bg_color)), popup_area);
+        }
+
         let block = Block::default()
             .borders(Borders::TOP | Borders::LEFT | Borders::RIGHT)
             .border_set(border::ROUNDED)
             .border_style(app.config.as_border_style())
-            .title_alignment(ratatui::prelude::Alignment::Center)
-            .title("Clear the queue?");
-        let text = Paragraph::new("Are you sure you want to clear the queue?")
-            .style(app.config.as_text_style())
-            .wrap(Wrap { trim: true });
+            .title_alignment(ratatui::prelude::Alignment::Center);
 
-        let popup_area = frame.area().centered_exact(45, 5);
-        frame.render_widget(Clear, popup_area);
+        let paragraph = Paragraph::new("Are you sure you want to clear the queue? This action cannot be undone.")
+            .style(app.config.as_text_style())
+            .wrap(Wrap { trim: true })
+            .block(block.clone())
+            .alignment(Alignment::Center);
 
-        if let Some(bg_color) = app.config.theme.modal_background_color {
-            frame.render_widget(Block::default().style(Style::default().bg(bg_color)), popup_area);
-        }
-        let [text_area, buttons_area] =
-            *Layout::vertical([Constraint::Length(2), Constraint::Max(3)]).split(popup_area)
+        let [content_area, buttons_area] =
+            *Layout::vertical([Constraint::Min(3), Constraint::Length(3)]).split(popup_area)
         else {
             return Ok(());
         };
 
-        frame.render_widget(
-            text,
-            block.inner(popup_area).inner(Margin {
-                horizontal: 1,
-                vertical: 0,
-            }),
-        );
-        frame.render_widget(block, text_area);
+        frame.render_widget(paragraph, content_area);
         frame.render_stateful_widget(&mut self.button_group, buttons_area, &mut self.button_group_state);
         Ok(())
     }
 
     fn handle_key(&mut self, key: &mut KeyEvent, client: &mut Client<'_>, context: &mut AppContext) -> Result<()> {
         if let Some(action) = key.as_common_action(context) {
             match action {
-                CommonAction::Down => {
+                CommonAction::Right => {
```

Comment:
What do you think about keeping both Right | Down and Left | Up?

Reply:
No real substantial reason, really. I initially put it to Up/Down as that effectively allowed to cycle through the whole list and both the buttons in the add to playlist modal with one button. Though having it on Right/Down only makes more sense I guess. Its up to you, Im fine with either way.

Reply:
I you don't mind, I prefer to keep it like this for now, if someone complains, we put back in the blink of an eye